test_scripts/Polices/External_UCS_Informing_/021_ATF_N_GetListofPermissions_empty_due_to_invalid_param_status_OnAppPermissionChange.lua

```diff
@@ -0,0 +1,126 @@
+---------------------------------------------------------------------------------------------
```

Comment:
@vvvakulenko, please update the folder of script. Now it's External_UCS_Informing_, it should be: External_UCS_Informing_HMI according to test set.